### PR TITLE
Issue 384 improve docs

### LINKED ISSUE #384: Fix missing content in Ocean.py py module
Is your feature request related to a problem? Please describe.
Read the docs parses the source code of `Ocean.py` and generates `.md` files. The generated markdowns are rendered here: https://docs.oceanprotocol.com/references/read-the-docs/ocean-py/. Some of the modules show No content present as there are no doc string the source code.

Describe the solution you'd like
Remove No content present by adding documentation in the source code.

### PR DESCRIPTION
**Pull request by akshay-ap**
Fixes #384 .

Changes proposed in this PR:

- Add docstrings

### CODE REVIEW

File: .docignore [hunk 1]
```diff
@@ -1 +1,2 @@
 **test**
+CHANGELOG.md
```
Comment (calina-c) [inline .docignore:2]: why is this file gitignored now?

**Review by calina-c — CHANGES_REQUESTED** [review]
Left a small question.
Could you also add type hinting wherever you added `:return:` docs, and no return type hint exists now?

File: .docignore (hunk 1 above)
Comment (akshay-ap) [reply .docignore:2]: We are not adding `CHANGELOG.md` file in docs.oceanprotcol.com. I had removed this file manually. Now, I added it in `.docignore` file.

**Comment by calina-c** [comment]
> Left a small question.
> Could you also add type hinting wherever you added `:return:` docs, and no return type hint exists now?

@akshay-ap I have a different PR for type hinting, so I'll merge this like it is.

**Review by alexcos20 — APPROVED** [review]

**Review by calina-c — APPROVED** [review]

**Pull request merged**